lemkus_spider.py

```diff
@@ -0,0 +1,59 @@
+import json
+import re
+import scrapy

```

Comment:
PEP8 violation. As told before, imports are grouped into three groups. `Scrapy` is not a built-in import.